avm/src/main.rs

```diff
@@ -26,19 +27,31 @@ fn main() {
   let matches = App::new(crate_name!())
     .version(crate_version!())
     // .author(crate_authors!(", ")) // Causes a warning, digging in clap's source it's not obvious
-    .about("The Alan Compiler and VM")
+    .about("Compile, run and deploy Alan")
     .subcommand(SubCommand::with_name("run")
       .about("Runs compiled .agc files")
       .version(crate_version!())
       // .author(crate_authors!(", "))
       .arg_from_usage("<FILE> 'Specifies the file to load'"))
     .subcommand(SubCommand::with_name("compile")
-      .about("Compiles the given source file (.ln, .amm, .aga) to a new output file (.amm, .aga, .agc, .js)")
+      .about("Compiles the given source file (.ln, .amm, .aga) to a new output file (.amm, .aga, .agz, .agc, .js)")
       .arg_from_usage("<INPUT> 'Specifies the input file to load'")
       .arg_from_usage("<OUTPUT> 'Specifies the output file to generate'"))
     .subcommand(SubCommand::with_name("install")
       .about("Install '/dependencies' from '.dependencies.ln'")
       .version(crate_version!()))
+    .subcommand(SubCommand::with_name("deploy-new")
+      .about("Deploys an .agz file to a new app in the cloud provider described in the deploy config at ~/.alan/deploy.json")
+      .arg_from_usage("<AGZ_FILE> 'Specifies the .agz file to deploy'"))
+    .subcommand(SubCommand::with_name("deploy-kill")
+      .about("Kills an app with the provided id in the cloud provider described in the deploy config at ~/.alan/deploy.json")
+      .arg_from_usage("<APP_ID> 'Specifies the alan app to kill'"))
+    .subcommand(SubCommand::with_name("deploy-status")
+      .about("Displays all the apps deployed in the cloud provider described in the deploy config at ~/.alan/deploy.json"))
+    .subcommand(SubCommand::with_name("deploy-upgrade")
+      .about("Deploys an .agz file to an existing app in the cloud provider described in the deploy config at ~/.alan/deploy.json")
+      .arg_from_usage("<APP_ID> 'Specifies the alan app to upgrade'")
+      .arg_from_usage("<AGZ_FILE> 'Specifies a source ln file to deploy'"))
```

Comment:
Should it be a source ln file that it then compiles, or an already compiled agz file as implied by the argument name?

Reply:
ah good catch

Reply:
It should match the rest